pkg/imgpkg/imageset/unprocessed_image_refs.go

```diff
@@ -11,37 +11,50 @@ import (
 	regname "github.com/google/go-containerregistry/pkg/name"
 )
 
+const RootBundleLabelKey string = "dev.carvel.imgpkg.copy.root-bundle"
```

Comment:
imageset does not know anything about root bundles. we should move this into a package that contains copy command (that's the only place where it's used afaik).

Reply:
ok moved to the `cmd` package